client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -35,6 +42,36 @@ export const getLocale = (language: SupportedLocales) => {
   }
 };
 
+const getLocalisations = (locale: typeof frFR) =>
+  locale.components.MuiLocalizationProvider.defaultProps.localeText;
+
+const getDateLocalisations = (language: SupportedLocales) => {
+  switch (language) {
+    case 'fr':
+    case 'fr-DJ':
+      return getLocalisations(frFR);
+
+    case 'es':
+      return getLocalisations(esES);
+
+    case 'ru':
+      return getLocalisations(ruRU);
+
+    case 'pt':
+      return getLocalisations(ptPT);
+
+    // Not every language is supported by MUI, and some dialects may want
+    // overrides. If/when needed - pass in t() here and set required fields,
+    // or define full localeText object for the required language
+    case 'en':
+    case 'ar':
+    case 'tet':
```

Comment:
Probably worth reviewing for tetum? Though haven't heard any concerns over existing english date picker 👀

Reply:
If it was gonna be anything I'd suggest `pt` as that's their other official language. Think fair to leave as en. I do wonder if it's worth checking with other PMs for changes for users above, as they aren't complaining either? With the exception of Russian as there are no Russian users yet.